ups/deploy_testing_distrib.table

```diff
@@ -0,0 +1,4 @@
+setupRequired(meas_deblender)
+setupRequired(obs_test)
+setupOptional(obs_cfht)
+setupOptional(obs_subru)
```

Comment:
All of the packages listed will pull in `afw`.  There's nothing wrong with that but it will result in a fairly long minimum build time.